src/app.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const { playRound } = require('./bullsAndCowsCalculator');
+const { generateSecretNumbers } = require('./numberGenerator');
+const { gameInput, gameOutput } = require('./gameIO');
+
+const secret = generateSecretNumbers();
+
+while (true) {
+  try {
+    const guess = gameInput();
+
+    const [isGameOver, resultOfRound] = playRound(guess, secret);
+
+    if (isGameOver) {
+      gameOutput('You win');
+      break;
+    }
+
+    gameOutput(resultOfRound);
```

Comment:
Output like `[ 'bull', 'cow', 'bull' ]` gives additional advantage to the player, he just needs to know the number of bulls/cows, e.g `Bulls: 2, cows: 1`